scripts/run-tests

```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+# expects cwd to be subpackage root
+
+NODE_ENV=test TZ='GMT' mocha \
+  --exclude "./node_modules/**" \
```

Comment:
would just `**/node_modules` work?

Reply:
tried it and nope - doesn't work

no idea why